awsmp/cli.py

```diff
@@ -358,6 +359,47 @@ def ami_product_release(product_id, offer_id):
     print(f'https://aws.amazon.com/marketplace/management/requests/{response["ChangeSetId"]}')
 
 
+@public_offer.command("add-instance-type")
+@click.argument("product-ids", nargs=-1)
+@click.option("--pricing-file-name", required=True)
+@click.option("--instance-type-file", required=True,
```

Comment:
After talking to Jess, I think we want to have a single file parameter here (not 2). That file should simply contain **all** the instance types (with prices) that we want to have enabled on the given product-id(s).
And then the code just adds the missing instance types.